languages/common-lisp/concepts/cons/links.json

```diff
@@ -1 +1,30 @@
-[]
+[
+  {
+    "url": "http://l1sp.org/cl/14",
+    "description": "Hyperspec Chapter on Conses"
+  },
+  {
+    "url": "http://l1sp.org/cl/cons",
```

Comment:
Does the chapter on conses contain things like the references for `cons`, `car`, etc? If so, I think we only need that link here. I've yet to formulate any rule for it, but I feel like the links here should be useful resources that haven't yet been mentioned in the `about.md`

Reply:
The chapter on conses is just the top level node which contains 'Cons Concepts' and 'Cons Dictionary'. The links to the specific functions point to the pages in the 'Dictionary' section.  I felt that these where *the* references for the concepts/functions. So I thought they go in this file?

Reply:
To be honest, I think I'd probably need to see how these are rendered on the website to form any sort of reliable opinion. In the meantime, I say we keep them in (as it's always easier to remove extra things that it is to remember what to add back) :)